organizations/views.py

```diff
@@ -26,12 +26,12 @@ class OrganizationViewSet(viewsets.ModelViewSet):
     @action(
         detail=True,
         permission_classes=[IsAuthenticated, OrganizationPermission],
-        methods=["put"],
+        methods=["patch"],
     )
     def setting(self, request, pk):
-        """Updates an org's setting key inside it's config"""
+        """Updates an org's settings key inside it's config"""
```

Comment:
similar to what we did for plio

```suggestion
        """Updates an org's settings"""
```